tap_zendesk_chat/http.py

```diff
@@ -12,35 +12,57 @@ class RateLimitException(Exception):
     pass
 
 
+class ClientGoneException(Exception):
+    pass
```

Comment:
nit: can we just log some info here so it's visible in the stack trace?

Reply:
Just changed it so that it passes in the contents of the error response as the exception message to give visibility into the original error.